### DISCUSSION TITLE
Personal website on github

### DISCUSSION THREAD

**Discussion by Variable-py** [community/community#29770] [Other Feature Feedback, Questions, & Ideas] [answered]
How can I build a personal website on github?
like http://amircfyt.github.io

**Comment by HiBixby** [comment, marked as answer]
Hello!
This is called "Github Pages"
you can see official docs from here -> https://pages.github.com/

to sum up, Create a new public repository(https://github.com/new) named username.github.io
(username is your username (or organization name))

and push to main branch
and you're done!
https://username.github.io will be created

**Reply by Variable-py** [reply]
Thank you bro♥
